prow/cmd/deck/job_history.go

```diff
@@ -67,7 +67,7 @@ type buildData struct {
 	Duration     time.Duration
 	Result       string
 	commitHash   string
-	Pull         *prowv1.Pull
+	Pulls        []prowv1.Pull
```

Comment:
wouldn't this break `.Pull` from https://github.com/kubernetes/test-infra/pull/22747/files?

Reply:
The code was changed accordingly 
https://github.com/kubernetes/test-infra/pull/22760/files#diff-5f8d6554e048de4637c1ba0e5ba42031d49e9dff2021510486955ebe338054b0R27-R33